app/components/primer/spinner_component.rb

```diff
@@ -31,13 +31,12 @@ class SpinnerComponent < Primer::Component
     # @param system_arguments [Hash] <%= link_to_system_arguments_docs %>
     def initialize(size: DEFAULT_SIZE, style: DEFAULT_STYLE, **system_arguments)
       @system_arguments = deny_tag_argument(**system_arguments)
-      @system_arguments[:tag] = :svg
+      @system_arguments[:tag] = :span
+      @system_arguments[:role] = :status
+      @system_arguments[:display] = :inline_block
```

Comment:
shouldn't a `span` always be `inline-block`?

Reply:
No they're `inline` by default, that's why the margin wasn't being respected

Reply:
hah TIL